tests/trainer/test_dataloaders.py

```diff
@@ -236,6 +236,51 @@ def test_multiple_dataloaders_passed_to_fit(tmpdir, ckpt_path):
         f'Multiple `test_dataloaders` not initiated properly, got {trainer.test_dataloaders}'
 
 
+@pytest.mark.parametrize(
+    ['train_percent_check', 'val_percent_check', 'test_percent_check'],
+    [
+        pytest.param(0.0, 0.0, 0.3),  # test percent check can never be 0
+        pytest.param(1.0, 1.0, 1.0),
+        pytest.param(0.2, 0.4, 0.4),
+    ]
+)
+def test_dataloaders_with_percent_check(tmpdir, train_percent_check, val_percent_check, test_percent_check):
+    """Verify num_batches for val & test dataloaders passed with percent_check"""
+    model = EvalModelTemplate()
```

Comment:
Yeah, fixes are merged, that's fine, but I still can't find this test to check the `number of batches` after we pass `limit_*_batches` in master.